media/css/m24/base-navigation-and-footer.scss

```diff
@@ -14,5 +14,6 @@
 @import 'vars/text';
 
 // global components
+@import 'transition';
 @import 'components/footer-refresh';
```

Comment:
Hmh while working this produces not so nice results for some pages that end with colour shapes, like:

> ![Screenshot 2024-09-22 at 17 43 03](https://github.com/user-attachments/assets/f4747be0-8e05-4fe5-b338-f5ba4cb00ef0)

(barely visible, but it's there…)

or better:

> ![Screenshot 2024-09-22 at 17 45 17](https://github.com/user-attachments/assets/cec31b20-b6d3-45d2-b690-316a9024b3a1)

Hiding the hr for now with low specificity `.m24-c-transition * { display: none; }` (that's only effective until the proper declaration is loaded) looks perhaps better — but that doesn't solve the issue once 2024 styles are turned on site-wide — when that divider gets styled globally and places with somewhat rectangular blocks ending the pages will display like this anyway…